webui.sh

```diff
@@ -3,20 +3,21 @@
 # Please do not make any changes to this file,  #
 # change the variables in webui-user.sh instead #
 #################################################
-
+CURRENT_DIR=$(cd $(dirname $0); pwd)
```

Comment:
```suggestion
cd $(dirname $0)
```
This is all you really need.

Reply:
Yes, your method is better, thanks!! In fact, my purpose is to achieve using systemctl on Ubuntu to run WebUI, remote server, direct call on SSH terminal webui.sh is not very stable, my Python is not very proficient, I need to learn more~

Reply:
In that case, you can set the working directory in the service file.
```
[Service]
WorkingDirectory=/path/to/directory
```